platform/security/idp/security-idp-server/src/main/java/org/codice/ddf/security/idp/server/IdpEndpoint.java

```diff
@@ -1159,8 +1142,8 @@ Response handleLogoutRequest(
     logoutState.setNameId(logoutRequest.getNameID().getValue());
```

Comment:
❓ This method re-assigns the `logoutState` parameter on L1140, which is bad practice and can become confusing. This is not something that was changed by this PR, but since re-assigning parameters is considered bad practice and can be confusing, should we consider declaring a new variable instead?

Reply:
Absolutely.